source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -101,3 +122,5 @@
      - ⊛
      - ✓
      - ✓
+
+.. [#v3-note] These driver versions are not compatible with MongoDB 8.1 because of an authentication issue. To learn more, see :ref:`java-server-8.1-incompatibility`.
```

Comment:
```suggestion
.. [#v3-note] These driver versions are not compatible with MongoDB 8.1 or later because of an authentication issue. To learn more, see :ref:`java-server-8.1-incompatibility`.
```